### PR TITLE
Add global flag exclude_stopwords (fixes issue #8)

### PR DESCRIPTION
**Pull request by AlexanderThaller**
The flag will enable stopwords filtering based on stop-words create.

The list of stopwords is importy with the crate stop-words. It contains
a list of ISO and NLTK stopwords and multiple languages.

Right now the default language is English.

The stopwords just get added to the existing exclude regexset. To make
the stopwords filter do what most people expect it will only match the
whole string and is configured to be case insentive.

Examples:

Without filter:

```
freq <<< "The bird is the word"

0.2 - 1 - The
0.2 - 1 - bird
0.2 - 1 - word
0.2 - 1 - is
0.2 - 1 - the
```

With filter:

```
freq --exclude-stopwords <<< "The bird is the word"

0.5 - 1 - word
0.5 - 1 - bird
```

### CODE REVIEW

File: src/bin/freq/main.rs [hunk 1]
```diff
@@ -37,7 +37,8 @@ fn run_main() -> Result<i32> {
     if let Some(c) = Config::load_from_file(&opts.config_file)? {
         opts.config.merge(c)
     }
-    let cfg = &opts.config;
+    let inputs = opts.inputs();
```
Comment (AlexanderThaller) [inline src/bin/freq/main.rs:41]: I did this and line 41 so I can take ownership of cfg without having partial move problems.

**Comment by mre** [comment]
Nice PR.
What do you think about adding a `stopwords` field to the `Exclude` struct here? https://github.com/mre/freq/blob/81c03f83856990f879c848f3673decba61b9a438/src/excludes.rs#L6-L8

Then inside of https://github.com/mre/freq/blob/81c03f83856990f879c848f3673decba61b9a438/src/excludes.rs#L17 we could add a check for the stopwords match after the regex. 

The advantage would be that we could later show the count of filtered stopwords separately from the manually excluded words via regex. That would be helpful for troubleshooting I guess?

We can also keep it as it is and change it later if we want.

**Comment by AlexanderThaller** [comment]
> Nice PR.
> What do you think about adding a `stopwords` field to the `Exclude` struct here?
> 
> https://github.com/mre/freq/blob/81c03f83856990f879c848f3673decba61b9a438/src/excludes.rs#L6-L8
> 
> Then inside of
> 
> https://github.com/mre/freq/blob/81c03f83856990f879c848f3673decba61b9a438/src/excludes.rs#L17
> we could add a check for the stopwords match after the regex.
> 
> The advantage would be that we could later show the count of filtered stopwords separately from the manually excluded words via regex. That would be helpful for troubleshooting I guess?
> 
> We can also keep it as it is and change it later if we want.

I was thinking of doing that but decided to do the smaller change instead. Will update the pullrequest!

**Review by mre — APPROVED** [review]

**Comment by mre** [comment]
Thanks! Looks great!

**Pull request merged**